component-library/src/app/pages/request-form/request-form.component.ts

```diff
@@ -0,0 +1,384 @@
+    title: 'RequestForm.Title',
+    anchorType: slugAnchorType.primary
+  };
+
+  requestCriteriaTitleSlugConfig: slugTitleURLConfig = {
+    type: slugTitleURLType.secondary,
+    title: 'RequestForm.RequestCriteriaTitle',
+    anchorType: slugAnchorType.primary
+  };
+
+  requestFormTitleSlugConfig: slugTitleURLConfig = {
+    type: slugTitleURLType.secondary,
+    title: 'RequestForm.RequestFormTitle',
+    anchorType: slugAnchorType.primary
+  };
+
+  bannerConfig: IBannerConfig = {
+    id: 'request-form-banner',
+    content: 'RequestForm.BannerDesc',
+    type: 'info',
+    rounded: false,
+    dismissible: false,
+    size: 'small',
+    cta: []
+  };
+
+  typeOfRequestRadioConfig: IRadioInputComponentConfig = {
+    id: 'radio-request-type',
+    formGroup: this.form,
+    label: 'RequestForm.typeOfRequestRadio',
+    required: true,
+    options: [
+      {
+        text: 'RequestForm.radioOption1',
+        value: 'Change'
+      },
+      {
+        text: 'RequestForm.radioOption2',
+        value: 'Request'
+      },
+      {
+        text: 'RequestForm.radioOption3',
+        value: 'Bug'
+      }
+    ],
+    errorMessages: [
+      {
+        key: 'required',
+        errorLOV: 'RequestForm.requiredRadioError'
+      }
+    ],
+    size: 'large',
+    disabled: false,
+    error: true
+  };
+
+  urgentRequestRadioConfig: IRadioInputComponentConfig = {
+    id: 'radio-request-urgent',
+    formGroup: this.form,
+    label: 'RequestForm.urgentRequestRadio',
+    required: true,
+    options: [
+      {
+        text: 'RequestForm.urgentOption1',
+        value: 'Y'
+      },
+      {
+        text: 'RequestForm.urgentOption2',
+        value: 'N'
+      }
+    ],
+    errorMessages: [
+      {
+        key: 'required',
+        errorLOV: 'RequestForm.requiredRadioError'
+      }
+    ],
+    size: 'large',
+    disabled: false,
+    error: true
+  };
+
+  requestTitleTextAreaConfig: ITextareaComponentConfig = {
+    formGroup: this.form,
+    id: 'request-title-text-area',
+    label: 'RequestForm.requestTitleLabel',
+    desc: 'RequestForm.requestTitleDesc',
+    required: true,
+    charLimit: '50',
+    resizable: 'vertical',
+    size: 'large',
+    errorMessages: [
+      {
+        key: 'required',
+        errorLOV: 'RequestForm.requiredTextAreaError'
+      },
+      {
+        key: 'maxLength',
+        errorLOV: 'RequestForm.fieldExceededMaxLength'
+      }
+    ]
+    // errorIcon?: IErrorIconConfig;
+  };
+
+  requestDetailsTextAreaConfig: ITextareaComponentConfig = {
+    formGroup: this.form,
+    id: 'request-details-text-area',
+    label: 'RequestForm.requestDetailsLabel',
+    desc: 'RequestForm.requestDetailsDesc',
+    required: true,
+    resizable: 'vertical',
+    size: 'large',
+    errorMessages: [
+      {
+        key: 'required',
+        errorLOV: 'RequestForm.requiredTextAreaError'
+      }
+      // {
+      //   key: 'maxLength',
+      //   errorLOV: 'RequestForm.fieldExceededMaxLength'
+      // }
```

Comment:
```suggestion
```

Reply:
Hi Mike - I can definitely remove those code comments. I just left them in there because the design isn't complete to include all the error states.